SmartPlaces.Facilities/lib/IngestionManager/src/IngestionManager.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<Sdk Name="Microsoft.Build.CentralPackageVersions" />
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+		<Version>$(VersionPrefix)</Version>
+		<Authors>Microsoft</Authors>
+		<Description>This is the Microsoft Smart Places Facilities Ingestion Manager library.</Description>
+		<AssemblyVersion>0.1.0</AssemblyVersion>
+		<PackageVersion>0.1.0-preview</PackageVersion>
+		<Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
```

Comment:
Is this still accurate? (Same question for all the file headers)

Reply:
Yes. I believe this is correct, as per the Microsoft open source standards.